opal/util/Makefile.am

```diff
@@ -121,12 +120,17 @@ libopalutil_core_la_SOURCES = \
         qsort.c \
         sha256.c \
         show_help.c \
-        show_help_lex.l \
+        show_help_content.c \
         stacktrace.c \
         string_copy.c \
         sys_limits.c \
         uri.c
 
+show_help_content.c: convert-help-files-to-c-code.py
+	$(OMPI_V_GEN) $(abs_srcdir)/convert-help-files-to-c-code.py \
```

Comment:
We really should call this as `$(OMPI_V_GEN) $(PYTHON) $(abs_srcdir)/convert-help-files-to-c-code.py`.  Shouldn't matter in the general case, but long term is slightly less assumy about where Python lives.